models/articles.models.js

```diff
@@ -26,7 +26,7 @@ exports.fetchArticleById = (article_id) => {
   if (isNaN(article_id)) {
```

Comment:
If the id is not a number then a psql error will be thrown and you could handle this with a PSQL error handler